platforms/core-configuration/declarative-dsl-core/src/main/resources/org/gradle/internal/declarativedsl/parsing/grammar.bnf

```diff
@@ -20,6 +20,7 @@
     COMA=','
     DOT='.'
     EQ='='
+    PLUSEQ='='
```

Comment:
❓ Should this be `+=` ?
```suggestion
    PLUSEQ='+='
```

Reply:
You are right, it should be `+=`. This grammar is just a reference, so a mistake here did not affect anything. Since this PR is already in the merge queue, I'll fix it in a follow-up PR.